src/site/stages/build/plugins/create-feature-toggles.js

```diff
@@ -0,0 +1,36 @@
+const {
+  FlipperClient,
+} = require('../../../../platform/utilities/feature-toggles/flipper-client.js');
+const environmentsConfigs = require('../../../constants/environments-configs');
+const environments = require('../../../constants/environments');
+
+function createFeatureToggles(buildOptions) {
+  const environmentName = buildOptions.buildtype;
+
+  return async (files, metalsmith, done) => {
+    const { fetchToggleValues } = FlipperClient({
+      host: environmentsConfigs[environmentName].API_URL,
+    });
+
+    let toggleValues;
+    try {
+      toggleValues = await fetchToggleValues();
+    } catch (error) {
+      if (environmentName === environments.LOCALHOST) {
+        toggleValues = {};
+      } else {
+        const errorMessage = `createFeatureToggles failed: ${error.message}`;
+        throw new Error(errorMessage);
+      }
+    }
+
+    Object.keys(files).forEach(file => {
+      // eslint-disable-next-line no-param-reassign
+      files[file].featureToggles = JSON.stringify(toggleValues);
```

Comment:
How do these get used in the build?

Also, do we want all the feature flags on all the pages? I don't see what harm it does, but I also don't know how this gets used.